openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx

```diff
@@ -73,7 +73,9 @@ const FeedListBody: FC<FeedListBodyProp> = ({
           };
           const postLength = feed.posts.length;
           const replies = feed.postsCount - 1;
-          const repliedUsers = feed.posts.map((f) => f.from).slice(0, 2);
+          const repliedUsers = feed.posts
+            .map((f) => f.from)
+            .slice(0, postLength === 3 ? 2 : 1);
```

Comment:
For postLength > 3, this will show only one user icon which is not ideal.

Reply:
@vivekratnavel , we only get 3 latest posts right?

Reply:
For now, we get 3 posts, but the length might change if the user replies. For instance, a thread can have 2 posts initially and when the user replies, I don't think we fetch the posts again. So, we should make sure we handle all the cases here to get a consistent UX. Also, please leave a comment on why we are slicing the posts here.

Reply:
@vivekratnavel , When we fetch a thread we only get 3 latest posts and when we add new replies that we get all the posts for that particular thread, but on the UI side we have some logic to get the latest 3 posts from that.

I'm slicing posts because we need to show the latest replied users avatat.